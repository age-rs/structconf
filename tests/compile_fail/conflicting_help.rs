//! Conflicting options for the help, which should only be available for
//! arguments.

use structconf::StructConf;

#[derive(StructConf)]
struct Config {
    #[conf(no_short, no_long, help = "out of place help message")]
    pub value: bool,
}

fn main() {}
