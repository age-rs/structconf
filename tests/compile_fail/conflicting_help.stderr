error: `no_short and no_long` and `help` are conflicting attributes, they cannot go together in the same field.
 --> $DIR/conflicting_help.rs:9:9
  |
9 |     pub value: bool,
  |         ^^^^^
